content/03-front-end-libraries/react/add-comments-in-jsx.md

```diff
@@ -1,42 +1,42 @@
 ---
 id: 5a24bbe0dba28a8d3cbd4c5e
-title: Add Comments in JSX
+title: Adicionando comentários em JSX
```

Comment:
I think it would be nice if you put "Adicione" instead of "Adicionando". When I read the titles on freeCodeCamp, I have a personal impression to read first verb in imperative tense... as commands lines. It's just a suggestion... 😊

Reply:
I'll fix these problems that you found. Thank's